distribution/zip/jballerina-tools/build.gradle

```diff
@@ -96,6 +99,13 @@ dependencies {
     dist "com.atomikos:atomikos-util:${project.atomikosUtilVersion}"
     dist "com.atomikos:transactions:${project.atomikosTransactionsVersion}"
 
+    // Following dependencies are required for testerina
+    dist "org.jacoco:org.jacoco.core:${project.jacocoVersion}"
+    dist "org.jacoco:org.jacoco.report:${project.jacocoVersion}"
+    dist "org.ow2.asm:asm-commons:${project.ow2AsmCommonsVersion}"
+    dist "io.github.java-diff-utils:java-diff-utils:${project.javaDiffUtilsVersion}"
+    jacocoAgentLib "org.jacoco:org.jacoco.agent:${project.jacocoVersion}:runtime"
```

Comment:
If we can remove `jacocoagent.jar` from the `lib` folder by adding this, can't we remove other added `asm` and `jacoco` jars from there?

Reply:
I have removed lib directory